packages/features/bookings/Booker/Booker.tsx

```diff
@@ -213,6 +213,10 @@ const BookerComponent = ({
     return setBookerState("booking");
   }, [event, selectedDate, selectedTimeslot, setBookerState]);
 
+  const slot = getQueryParam("slot");
+  useEffect(() => {
+    setSelectedTimeslot(slot || null);
```

Comment:
This makes sure that the `selectedTimeslot` updates in the store when it is changed in the query param. And it sets it to null when the `slot` query param is not present